about.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About - Psychological therapy Week Event 2023</title>
+  <link rel="icon" type="image/vnd.icon" href="./assets/img/favicon.ico">
+  <link rel="stylesheet" href="./assets/css/general.css">
+  <link rel="stylesheet" href="./assets/css/main.css">
+  <link rel="stylesheet" href="./assets/css/responsive.css">
+</head>
+
+<body>
+  <section id="mobile-menu" class="hidden box_flex flex_col justify_center gap_40">
+    <ul class="box_flex flex_col align_center gap_40">
+      <li><a href="index.html" class="mobile-nav-link" data-name="Home">Home</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Program">Program</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Join">Join</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="Sponsor">Sponsor</a></li>
+      <li><a href="#" class="mobile-nav-link" data-name="News">News</a></li>
+    </ul>
+    <div class="box_flex justify_center menu-footer-button">
+      <a href="#" class="btn_default primary_line_button">MH Campaigns</a>
+    </div>
+  </section>
+  <header id="header">
+    <div class="banner show-banner">
+      <div class="container">
+        <ul class="box_flex justify_end align_center gap_20">
+          <li><a href="#"><img src="./assets/img/facebook.png" alt="Facebook icon in white color"></a></li>
+          <li><a href="#"><img src="./assets/img/twitter.png" alt="Twitter icon in white color"></a></li>
+          <li><a href="#" class="text_white">French</a></li>
+          <li><a href="#" class="text_white">Login</a></li>
+        </ul>
+      </div>
+    </div>
+    <div id="navbar">
+      <div class="container">
+        <div class="box_flex align_center" id="navbar-content">
+          <div class="btn-mobile-menu">
+            <button type="button" class="btn_default box_flex flex_col align_center" id="open-mobile-menu">
+              <span></span>
+              <span></span>
+              <span></span>
+            </button>
+          </div>
+          <div class="logo-container">
+            <a href="index.html">
+              <img src="./assets/img/logo-original.png" alt="Logo Psychological Therapy Week Event 2023">
+            </a>
+          </div>
+          <nav class="navbar-menu">
+            <ul class="box_flex align_center">
+              <li><a href="about.html" class="nav-link">About</a></li>
```

Comment:
On the desktop version, kindly include navigation to the home page. This will improve user experience.

![image](https://github.com/NitBravoA92/capstone-project/assets/20554583/e4a9ffb0-f2dc-467f-98dc-37b4325a67c5)